WebRoot/css/zm.css

```diff
@@ -1667,7 +1667,31 @@ div.CompDragArea * {
 
 .ZmGrantRightsDialog .addrBubbleContainer,
 .ZmPreferencesPage .addrBubbleContainer {
-    width: 290px;
+	@GrantRightsDialogInputContainer@
+}
+
+.ZmGrantRightsDialog .WindowOuterContainer {
+	@GrantRightsDialogOuterContainer@
+}
+
+.ZmGrantRightsDialog #ZmGrantRightsDialog_name_row .Label {
+	@GrantRightsDialogEmailLabel@
+}
+
+.ZmGrantRightsDialog INPUT[type='text'] {
+	@GrantRightsDialogInput@
+}
+
+.ZmGrantRightsDialog .Label:empty {
+	display:none;
+}
+
+.ZmGrantRightsDialog .SendAs, .ZmGrantRightsDialog .SendObo {
+	padding-top:@SkinWrapperPadding@;
```

Comment:
I briefly looked at Gmail and Face book settings page, where the layout is more or less same as ours  and they are using padding to bring similar to this.